### ISSUE TITLE
Still alive ?

### ISSUE THREAD

**Issue by Ippephyx** [alexylem/jarvis#966]
### Description
Bonjour, j'aurais juste voulu savoir si le projet est toujours d'actualité et encore développé mis-à-jour ou est-il en phase d'abandon (2 ans depuis la dernière mis à jour...). Merci !

Hi, I just wanted to know if the project is still live and updated or maybe it has been abandonned ? (2 years since last commit). Thanks !

### Réponse / Answer
**Pour répondre à ma question :**
D'après @Oliv4945 le projet serait effectivement abandonné.
Personnellement, je vais essayer d'obtenir une autorisation de _forker_ Jarvis.
--
**To answer the question :** 
From what @Oliv4945 could gather, it would seem that this project is _sadly_ abandonned.
As for myself, I will try to get permission to fork Jarvis.

**Comment by Oliv4945** [comment]
Non, j'ai écris plusieurs émail sans réponse a @alexylem , je vais l'archiver...
No, je project is notre maintained anymore

**Comment by Oliv4945** [comment]
Je n'ai pas les droits 😃 espérons que cette issue suffise a guider les gens

**Comment by Ippephyx** [comment]
> 
> 
> Je n'ai pas les droits 😃 espérons que cette issue suffise a guider les gens

En fait je voulais contacter @alexylem afin de lui demander l’autorisation de _"forker"_ Jarvis afin d'essayer de moderniser tout ça et commencer à rendre ce nouveau fork plus au "goût du jour".

I wanted to contact @alexylem to ask him permission to _fork_ Jarvis in order to modernise it and begin to build a more "up to date" project upon Jarvis.

**Comment by Oliv4945** [comment]
Pourquoi faire un fork et non des pull request ?

**Comment by steveoriol** [comment]
Qui va approuver les "pull request" si le projet est à l'abandon...

**Comment by Oliv4945** [comment]
Je n'ai plus le temps de chercher les bugs/améliorations ni de répondre à beaucoup d'issues, mais je peux merger une PR ;-)

**Comment by Sispheor** [comment]
Hello hello. ça faisait un moment que j'étais pas passé sur le projet. Dommage que il soit à l'arret.
Il est peut etre possible de merger nos communautés?
Du coté de [Kalliopé](https://github.com/kalliope-project/kalliope) on est tjs en activité.

En tout cas bonne continuation à vous !

**Comment by ferrader** [comment]
Hello,
I work in CNRS and I propose to fork this project because I need this for my work.
Here the github I use https://github.com/ferrader/openjarvis_nextgen.
Thanks you
